ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.12.yaml

```diff
@@ -136,6 +136,15 @@ tests:
         group\|The OpenStack platform creates Control plane nodes in a server group
       TEST_SUITE: experimental/reliability/minimal
     workflow: openshift-e2e-openstack-upi
+- as: e2e-openstack-appcreds
+  interval: 24h
+  steps:
+    cluster_profile: openstack-vexxhost
+    env:
+      APPLICATION_CREDENTIALS: "true"
+      # these tests are skipped since they use cinderclient which does not support application credentials
```

Comment:
Prow doesn't like comments in config.

Running `make ci-operator-config` (part of `make update`) removes them.

Reply:
Done